platforms/chibios/boards/STEMCELL/convert_to_stemcell.mk

```diff
@@ -0,0 +1,14 @@
+# Proton C MCU settings for converting AVR projects
+TARGET := $(TARGET)_stemcell
+MCU := STM32F411
+BOARD := STEMCELL
+BOOTLOADER := tinyuf2
+OPT_DEFS += -DCONVERT_TO_STEMCELL
+MCU_LDSCRIPT := STEMCELL_tinyuf2
```

Comment:
Should it always require tinyuf2? Are they coming with it preloaded? Do people need to flash it as a bootloader first?
Given that DFU isn't problematic for F4xx, should the default still be DFU, with UF2 as an option?

Reply:
Yes, STeMCell does not have fancy circuit for reset/bootloader logic. 
The `reset` pinout is directly connected to STM32 NRST pin and Boot0 pin is connected to onboard switch.

With this setup, the normal reset button in keyboard, can only reset. To go to DFU, one has to click Boot0 and then reset button.
Since most of the keyboards will have controllers face down or have OLEDs on top, will be inconvenient to use STM32 inbuilt DFU
TinyUF2 as default, will remove this nuisance and regular user can still single click for reset and double click for bootloader just like promicro

STeMCell after PCBA does not have tiny-uf2 bootloader, We need to load the tinyuf2 bootloader from below using STM32 DFU bootloader.
https://github.com/megamind4089/STeMCell/tree/main/bootloader